src/lib/controllers/ProvisioningController.js

```diff
@@ -22,6 +23,10 @@ class ProvisioningController extends Controller {
     coreID: string,
     postBody: { publicKey: string },
   ): Promise<*> {
+    if (!postBody.publicKey) {
```

Comment:
I think it's more logical to check this in controller level. Moreover before it was checking after call ursa.createPublicKey() -this was unexpected behavior and throws error like 'can't do toString() of undefined'